src/states/common/approved.rs

```diff
@@ -170,6 +171,17 @@ pub trait Approved: Relocated {
                     trace!("{} Parsec Remove: - {}", self, peer_id);
                     self.chain_mut().handle_churn_event(&event, proof_set)?;
                 }
+                obs @ Observation::StartDkg(_) | obs @ Observation::DkgMessage(_) => {
+                    log_or_panic!(
+                        LogLevel::Error,
+                        "parsec_poll polled internal Observation: {:?}",
+                        obs
+                    );
+                }
+                Observation::DkgResult { .. } => {
+                    // FIXME: Handle properly
```

Comment:
shall we use `TODO` ?
btw. any policy here that which key word we shall choose?

Reply:
I'll just remove it (it was a copy paste).